ext/zip/php_zip.stub.php

```diff
@@ -721,6 +720,27 @@ class ZipArchive implements Countable
      */
     public const LIBZIP_VERSION = UNKNOWN;
 
+    /* since 1.10.1 */
+#ifdef ZIP_LENGTH_TO_END
+    /**
+     * @var int
+     * @cvalue ZIP_LENGTH_TO_END
+     */
+    public const LENGTH_TO_END = UNKNOWN;
+#else
+    /**
+     * @var int
+     */
+    public const LENGTH_TO_END = 0;
```

Comment:
please avoid declaring the same constant "twice" when possible as doing so may introduce problems to the generation of the manual AFAIR. E.g. You could conditionally declare `ZIP_LENGTH_TO_END ` with the default value in the header file.

Reply:
done